WordPress/gutenberg
# Update Post Link block variation icon to match the one used in the Link UI results.

## What?


Update Post Link block variation icon to match the one used in the Link UI results.

Fixes https://github.com/WordPress/gutenberg/issues/50181

## Testing Instructions





1. Go to the site editor.
2. Edit the header, and select the navigation block.
3. Add a new post link.
4. Confirm that the icon is the same as the one used in the link UI results.

## Screenshots or screencast 

https://user-images.githubusercontent.com/252415/235202801-36f76f0e-e228-43e8-960d-39552251adcd.mov
